src/Symfony/Component/TypeInfo/README.md

```diff
@@ -0,0 +1,14 @@
+TypeInfo Component
+==================
+
+The TypeInfo component extracts information about PHP types using
+metadata of popular sources.
```

Comment:
We're moving docs for standalone usage from symfony-docs to each component's README, it'd be nice to have some snippet here too. Check [password-hasher](https://github.com/symfony/password-hasher?tab=readme-ov-file#getting-started) or [error-handler](https://github.com/symfony/error-handler?tab=readme-ov-file#getting-started) for inspiration